### PR TITLE
DEVELOPER-4741 Created New RHDP Theme

### PR DESCRIPTION
**Pull request by adelasofia**
[DEVELOPER-4741 Create New RHDP Theme](https://issues.jboss.org/browse/DEVELOPER-4741)

### CODE REVIEW

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.libraries.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.1.0/custom-elements-es5-adapter.js: { type: external, minified: true }
+    https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.1.0/webcomponents-lite.js: { type: external, minified: true }
+
+base-theme:
+  # This should be all of the style sheets broken up according to smaccs
+  css:
+    base:
+      css/base/app.css: {}
+  js:
+    https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.3/angular.min.js: { type: external, minified: true }
```
[the remark this answers is not in the document]
Comment (KamiQuasi) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.libraries.yml]: I'm okay to leave them for now as the entirety of the javascripts directory will be able to be scrapped once we migrate to the external repo for front-end assets.

**Comment by adelasofia** [comment]
@LightGuard I've made the fixes you requested

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+ * our site css so we could get the responsive video to work.
+ *
+ * Adding another edit for embedded products.
+ *
+ * @param $variables
+ * @param $hook
+ */
+function rhdp_preprocess_field(&$variables, $hook) {
+    if ($variables['field_type'] === 'video_embed_field') {
+        $variables['items'][0]['content']['#attached']['library'][] = 'rhd/video_embed_field.responsive-video';
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhdp.theme]: I think the path here needs to be fixed to be rhdp

**Comment by adelasofia** [comment]
retest this please

**Comment by adelasofia** [comment]
retest this please

**Comment by LightGuard** [comment]
retest this please

**Comment by KamiQuasi** [comment]
retest this please

**Pull request merged**